pkg/sdk/authentication_policies_impl_gen.go

```diff
@@ -0,0 +1,153 @@
+package sdk
+
+import (
+	"context"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/collections"
+)
+
+var _ AuthenticationPolicies = (*authenticationPolicies)(nil)
+
+type authenticationPolicies struct {
+	client *Client
+}
+
+func (v *authenticationPolicies) Create(ctx context.Context, request *CreateAuthenticationPolicyRequest) error {
+	opts := request.toOpts()
+	return validateAndExec(v.client, ctx, opts)
+}
+
+func (v *authenticationPolicies) Alter(ctx context.Context, request *AlterAuthenticationPolicyRequest) error {
+	opts := request.toOpts()
+	return validateAndExec(v.client, ctx, opts)
+}
+
+func (v *authenticationPolicies) Drop(ctx context.Context, request *DropAuthenticationPolicyRequest) error {
+	opts := request.toOpts()
+	return validateAndExec(v.client, ctx, opts)
+}
+
+func (v *authenticationPolicies) Show(ctx context.Context, request *ShowAuthenticationPolicyRequest) ([]AuthenticationPolicy, error) {
+	opts := request.toOpts()
+	dbRows, err := validateAndQuery[showAuthenticationPolicyDBRow](v.client, ctx, opts)
+	if err != nil {
+		return nil, err
+	}
+	resultList := convertRows[showAuthenticationPolicyDBRow, AuthenticationPolicy](dbRows)
+	return resultList, nil
+}
+
+func (v *authenticationPolicies) ShowByID(ctx context.Context, id SchemaObjectIdentifier) (*AuthenticationPolicy, error) {
+	authenticationPolicies, err := v.Show(ctx, NewShowAuthenticationPolicyRequest())
```

Comment:
You have to add additional filtering to the request (see other ShowByID implementations). For schema objects, ideally, a combination of `like` and `in schema` would be great (to limit the show output as much as possible).